recipes/jgo/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "jgo" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 880d880badc2b3ec88ed15f063be3fe12c83eec5c951b743b182b66d81977188
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vvv"
+  skip: True  # [py2k]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+
+test:
+  imports:
+    - jgo
+
+about:
+  home: https://github.com/scijava/jgo
+  license: PUBLIC-DOMAIN
+  license_family: PUBLIC-DOMAIN
+  summary: 'Launch Java code from the CLI, installation-free. '
+  doc_url: https://github.com/scijava/jgo
+  dev_url: https://github.com/scijava/jgo
+
+extra:
+  recipe-maintainers:
+    - ctrueden
+    - hanslovsky
```

Comment:
@hanslovsky @ctrueden are you OK with being maintainers?

Reply:
I am OK with that.